test/integration/api/app-users.js

```diff
@@ -220,7 +220,7 @@ describe('api: /projects/:id/app-users', () => {
       service.login('alice', (asAlice) =>
         asAlice.post('/v1/projects/1/app-users').send({ displayName: 'condemned' }).expect(200)
           .then(({ body }) => asAlice.delete('/v1/projects/1/app-users/' + body.id).expect(200))
-          .then(() => asAlice.get('/v1/audits')
+          .then(() => asAlice.get('/v1/audits?action=field_key')
```

Comment:
After the addition of `user.session.create`, a number of tests related to the audit log started failing. Where possible, I specify the `action` filter in the test, which seemed like the easiest way to update the test.